Code/GraphMol/MolDraw2D/Wrap/rdMolDraw2D.cpp

```diff
@@ -543,6 +547,17 @@ void setDrawerColour(RDKit::MolDraw2D &self, python::tuple tpl) {
   self.setColour(pyTupleToDrawColour(tpl));
 }
 
+#ifdef RDK_BUILD_QT_SUPPORT
+MolDraw2DQt *moldrawFromQPainter(int width, int height, unsigned long ptr,
```

Comment:
why are we not passing ptr as QPointer*?

Reply:
On the Python side we just have the PyQt wrapped object and we can't get the QPointer* from that. The best we can do, as far as I can tell, is get the pointer as an int using sip.unwrapobject and pass that ,